tools/toolchain/scripts/install_libvdwxc.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash -e
+[ "${BASH_SOURCE[0]}" ] && SCRIPT_NAME="${BASH_SOURCE[0]}" || SCRIPT_NAME=$0
+SCRIPT_DIR="$(cd "$(dirname "$SCRIPT_NAME")" && pwd -P)"
+libvdwxc_ver=${libvdwxc_ver:-master-429a80027a2ec2c97e2f6d9a3dc84843f2739865}
+
+source "${SCRIPT_DIR}"/common_vars.sh
+source "${SCRIPT_DIR}"/tool_kit.sh
+source "${SCRIPT_DIR}"/signal_trap.sh
+
+with_libvdwxc=${1:-__INSTALL__}
+
+[ -f "${BUILDDIR}/setup_libvdwxc" ] && rm "${BUILDDIR}/setup_libvdwxc"
+
+if [ "$MPI_MODE" = "no" ] && [ "$ENABLE_OMP" = "__FALSE__" ] && [ $with_sirius ="__FALSE__" ] ; then
+    report_warning $LINENO "MPI and OpenMP and SIRIUS are disabled, skipping libvdwxc installation"
+    exit 0
+fi
+
+! [ -d "${BUILDDIR}" ] && mkdir -p "${BUILDDIR}"
+cd "${BUILDDIR}"
+case "$with_libvdwxc" in
+    __INSTALL__)
+        require_env MPI_CFLAGS
+        require_env MPI_LDFLAGS
+        require_env MPI_LIBS
+        require_env FFTW_LDFLAGS
+        require_env FFTW_LIBS
+        require_env FFTW_CFLAGS
+
+        echo "==================== Installing libvdwxc ===================="
+        pkg_install_dir="${INSTALLDIR}/libvdwxc-${libvdwxc_ver}"
+        install_lock_file="$pkg_install_dir/install_successful"
+        if verify_checksums "${install_lock_file}" ; then
+            echo "libvdwxc-${libvdwxc_ver} is already installed, skipping it."
+        else
+            if [ -f libvdwxc-${libvdwxc_ver}.tar.gz ] ; then
+                echo "libvdwxc-${libvdwxc_ver}.tar.gz is found"
+            else
+                # do not remove this. They do not publish official version often
+                download_pkg ${DOWNLOADER_FLAGS} \
+                             https://www.cp2k.org/static/downloads/libvdwxc-${libvdwxc_ver}.tar.gz
+            fi
+
+            for patch in "${patches[@]}" ; do
+                fname="${patch##*/}"
+                if [ -f "${fname}" ] ; then
+                    echo "${fname} is found"
+                else
+                    # parallel build patch
+                    download_pkg ${DOWNLOADER_FLAGS} "${patch}"
+                fi
+            done
+
+            echo "Installing from scratch into ${pkg_install_dir}"
+            [ -d libvdwxc-${libvdwxc_ver} ] && rm -rf libvdwxc-${libvdwxc_ver}
+            tar -xzf libvdwxc-${libvdwxc_ver}.tar.gz
+            cd libvdwxc-${libvdwxc_ver}
+
+            for patch in "${patches[@]}" ; do
+                patch -p1 < ../"${patch##*/}"
+            done
+            unset MPICC MPICXX MPIF90 MPIFC MPIF77
+            ./autogen.sh
```

Comment:
Could you please redirect this too. I believe it's responsible for the following messages:
```
Generating aclocal.m4...
done.
Generating config.h.in...
done.
Generating configure script...
done.
Generating libtool scripts...
done.
Generating Makefile.in for each directory...
configure.ac:348: installing 'config/gnu/compile'
configure.ac:23: installing 'config/gnu/config.guess'
configure.ac:23: installing 'config/gnu/config.sub'
configure.ac:24: installing 'config/gnu/install-sh'
configure.ac:24: installing 'config/gnu/missing'
src/Makefile.am: installing 'config/gnu/depcomp'
done.
```